_data/authors.yaml

```diff
@@ -209,4 +209,10 @@ dplatz:
   emailhash: "0ae56c62d1b2142ba2868725b956b7ab"
   job_title: "Software Engineer"
   twitter: "38leinaD"
-  bio: "Developer/consultant in the banking sector. Quarkus enthusiast."
\ No newline at end of file
+  bio: "Developer/consultant in the banking sector. Quarkus enthusiast."
+  jbeck:
+    name: "Jeff Beck"
+    email: "jebeck@redhat.com"
```

Comment:
this email does not seem registered on gravatar.com  it shows up empty

Reply:
it's the email hash below that should be used for gravatar AFAIK

Reply:
in the end it wasn't  the email that was missing but bad indent in the yaml caused it to "not exist'. fixed before merging.